src/5e-SRD-Features.json

```diff
@@ -1117,107 +759,207 @@
       "name": "Bard",
       "url": "/api/classes/bard"
     },
-    "name": "Choose: Expertise 2",
+    "name": "Expertise 2",
```

Comment:
This could be something for a different PR, and I know it's not something you've introduced in this PR, but I'm wondering whether names like "Expertise (10th Level)" would be more appropriate than "Expertise 2"? We could also cut down the description to only include the effects that happen at this level, perhaps?

Reply:
I guess that the change is small for now, and we should keep as is. We can discuss this in greater lengths, see #382 for that. Adding this request would increase the inconsistency or, if done entirely, add more complexity to this PR.

Reply:
Yeah. I think keeping it as is for now makes sense and we can make a separate PR to address the discussion in #382